pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UsePatternVariableTest.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.bestpractices;
+
+import net.sourceforge.pmd.test.PmdRuleTst;
+
+class UsePatternVariableTest extends PmdRuleTst {
+    // no additional unit tests
```

Comment:
is this any good ? 

@adangel @oowekyala feel free to push implementation real quick ^^, assuming its a ease for you guys.